networking/network_security/configuring-ipsec-ovn.adoc

```diff
@@ -6,26 +6,30 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-With IPsec enabled, you can encrypt both internal pod-to-pod cluster traffic between nodes and external traffic between pods and IPsec endpoints external to your cluster. All pod-to-pod network traffic between nodes on the OVN-Kubernetes cluster network is encrypted with IPsec in _Transport mode_.
+By enabling IPsec, you can encrypt both internal pod-to-pod cluster traffic between nodes and external traffic between pods and IPsec endpoints external to your cluster. All pod-to-pod network traffic between nodes on the OVN-Kubernetes cluster network is encrypted with IPsec in _Transport mode_.
 
-IPsec is disabled by default. It can be enabled either during or after installing the cluster. For information about cluster installation, see xref:../../installing/overview/index.adoc#ocp-installation-overview[{product-title} installation overview].
+IPsec is disabled by default. You can enable IPsec either during or after installing the cluster. For information about cluster installation, see xref:../../installing/overview/index.adoc#ocp-installation-overview[{product-title} installation overview].
+
+The following support limitations exist for IPsec on a {product-title} cluster:
+
+* On {ibm-cloud-name}, IPsec supports only NAT-T. Encapsulating Security Payload (ESP) is not supported on this platform.
+* On {vmw-first}, using ESP hardware offloading after enabling IPsec is not supported. ESP hardware offloading is automatically enabled, so you must disable it if you want to enable IPsec for your cluster.
+* If your cluster uses link:https://www.redhat.com/en/topics/containers/what-are-hosted-control-planes[{hcp}] for Red Hat {product-title}, IPsec is not supported for IPsec encryption of either pod-to-pod or traffic to external hosts.
```

Comment:
Some notes:
* This applies everywhere, not only vmware
* OC will disable ESP offload in OCP managed interfaces so that is not a problem. However the user needs to make sure ESP offload is not used in other non OCP managed interfaces